yarn.lock

```diff
@@ -1509,7 +1509,7 @@
   resolved "https://registry.yarnpkg.com/@babel/compat-data/-/compat-data-7.28.0.tgz#9fc6fd58c2a6a15243cd13983224968392070790"
   integrity sha512-60X7qkglvrap8mn1lh2ebxXdZYtUcpd7gsmy9kLaBJ4i/WdY8PqTSdxyA8qraikqKQK5C1KRBKXqznrVapyNaw==
 
-"@babel/core@7.28.0", "@babel/core@^7.0.0", "@babel/core@^7.18.9", "@babel/core@^7.21.3", "@babel/core@^7.27.4", "@babel/core@^7.27.7", "@babel/core@^7.28.0":
+"@babel/core@7.28.0":
```

Comment:
If we're adding a new babel version, do we want to try and unify all the versions?